docs/USAGE-GUIDE.md

```diff
@@ -1511,6 +1511,71 @@ int s2n_cert_get_utf8_string_from_extension_data(const uint8_t *extension_data,
 
 **s2n_cert_get_utf8_string_from_extension_data** gets the UTF8 String representation of the DER encoded ASN.1 X.509 certificate extension data.
 
+### S2N Get Selected Handshake Algorithms
+
+`s2n_tls_signature_algorithm` is defined as:
+
+```c
+typedef enum {
+    S2N_TLS_SIGNATURE_ANONYMOUS = 0,
+    S2N_TLS_SIGNATURE_RSA = 1,
+    S2N_TLS_SIGNATURE_ECDSA = 3,
+
+    /* Use Private Range for RSA PSS since it's not defined there */
+    S2N_TLS_SIGNATURE_RSA_PSS_RSAE = 224,
+    S2N_TLS_SIGNATURE_RSA_PSS_PSS
+} s2n_tls_signature_algorithm;
+```
+
+### s2n\_connection\_get\_selected\_signature\_algorithm
+
+```c
+int s2n_connection_get_selected_signature_algorithm(struct s2n_connection *conn, s2n_tls_signature_algorithm *chosen_alg);
+```
+
+**s2n_connection_get_selected_signature_algorithm** gets the negotiated signature algorithm for the connection. 
+
```

Comment:
Should we mention what behavior to expect before a signature algorithm has been negotiated? Same for the client version: what if we aren't doing client auth?